sgx-dcap-ql-sys/build.rs

```diff
@@ -0,0 +1,9 @@
+use std::env;
+
+fn main() {
+    let bindings: bool = !env::var("CARGO_FEATURE_BINDINGS").map_or(false, |val| val == "true");
+
+    if bindings {
+        // println!("cargo:rustc-link-lib=sgx_dcap_ql");
```

Comment:
I saw you commented this out when you were adding the quote generations. Is this meant to stay commented out?

Reply:
Good catch!  I am fairly certain this needs to be uncommented, since `sgx_dcap_ql` is listed under the `package.links`. Refer to the [relevant cargo documentation](build-scripts).

```suggestion
        println!("cargo:rustc-link-lib=sgx_dcap_ql");
```

[build-scripts]: https://doc.rust-lang.org/cargo/reference/build-scripts.html#the-links-manifest-key

Reply:
Your "relevant cargo documentation" link leads to a 404 page. Could you please relink it? :)